test/bun.js/install/bun-install.test.ts

```diff
@@ -188,6 +189,51 @@ it("should handle workspaces", async () => {
   await access(join(package_dir, "bun.lockb"));
 });
 
+it("should handle workspaces with packages array", async () => {
+  rmSync(join(package_dir, "./bun.lockb"), { recursive: true, force: true });
```

Comment:
`package_dir` should be randomly generated, empty directory - did you run into a failure here somehow? :thinking:

https://github.com/oven-sh/bun/blob/e7cd45d0fa3f381238ea2ebfbba2d9e738376fcc/test/bun.js/install/dummy.registry.ts#L80